src/content/docs/fr/reference/integrations-reference.mdx

```diff
@@ -0,0 +1,727 @@
+
+- **Renderers :** [`lit`](/fr/guides/integrations-guide/lit/), [`svelte`](/fr/guides/integrations-guide/svelte/), [`react`](/fr/guides/integrations-guide/react/), [`preact`](/fr/guides/integrations-guide/preact/), [`vue`](/fr/guides/integrations-guide/vue/), [`solid`](/fr/guides/integrations-guide/solid-js/)
+- **Bibliothèques :** [`tailwind`](/fr/guides/integrations-guide/tailwind/), [`partytown`](/fr/guides/integrations-guide/partytown/)
+- **Caractéristiques :** [`sitemap`](/fr/guides/integrations-guide/sitemap/)
+
+## Référence rapide de l'API
+
+```ts
+interface AstroIntegration {
+  name: string;
+  hooks: {
+    'astro:config:setup'?: (options: {
+      config: AstroConfig;
+      command: 'dev' | 'build' | 'preview';
+      isRestart: boolean;
+      updateConfig: (newConfig: DeepPartial<AstroConfig>) => AstroConfig;
+      addRenderer: (renderer: AstroRenderer) => void;
+      addWatchFile: (path: URL | string) => void;
+      addClientDirective: (directive: ClientDirectiveConfig) => void;
+      addMiddleware: (middleware: AstroIntegrationMiddleware) => void;
+      addDevToolbarApp: (pluginEntrypoint: string) => void;
+      injectScript: (stage: InjectedScriptStage, content: string) => void;
+      injectRoute: ({ pattern: string, entrypoint: string }) => void;
+      logger: AstroIntegrationLogger;
+    }) => void | Promise<void>;
+    'astro:config:done'?: (options: { config: AstroConfig; setAdapter: (adapter: AstroAdapter) => void; logger: AstroIntegrationLogger; }) => void | Promise<void>;
+    'astro:server:setup'?: (options: { server: vite.ViteDevServer; logger: AstroIntegrationLogger; }) => void | Promise<void>;
+    'astro:server:start'?: (options: { address: AddressInfo; logger: AstroIntegrationLogger; }) => void | Promise<void>;
+    'astro:server:done'?: (options: { logger: AstroIntegrationLogger; }) => void | Promise<void>;
+    'astro:build:start'?: (options: { logger: AstroIntegrationLogger; }) => void | Promise<void>;
+    'astro:build:setup'?: (options: {
+      vite: ViteConfigWithSSR;
+      pages: Map<string, PageBuildData>;
+      target: 'client' | 'server';
+      logger: AstroIntegrationLogger;
+    }) => void | Promise<void>;
+    'astro:build:generated'?: (options: { dir: URL; logger: AstroIntegrationLogger; }) => void | Promise<void>;
+    'astro:build:ssr'?: (options: {
+        manifest: SerializedSSRManifest;
+        entryPoints: Map<RouteData, URL>;
+        logger: AstroIntegrationLogger;
+    }) => void | Promise<void>;
+    'astro:build:done'?: (options: { dir: URL; routes: RouteData[]; logger: AstroIntegrationLogger; }) => void | Promise<void>;
+  };
+}
+```
+
+## Hooks
+
+### `astro:config:setup`
+
+**Prochain hook :** [`astro:config:done`](#astroconfigdone)
+
+**Quand :** Lors de l'initialisation, avant que les configurations de [Vite](https://vitejs.dev/config/) ou d'[Astro config](/fr/reference/configuration-reference/) ne soient résolues.
```

Comment:
My mistake, I updated my comment... you probably missed the update. You should remove `config` from the anchor since we have `configurations` before.

`Astro config` => `Astro`